stable/datadog/templates/deployment.yaml

```diff
@@ -1,25 +1,37 @@
+{{- if .Values.deployment.enabled }}
 {{- if .Values.datadog.apiKey }}
-{{- if .Values.datadog.collectEvents }}
 apiVersion: extensions/v1beta1
 kind: Deployment
 metadata:
-  name: {{ template "fullname" . }}-events
+  name: {{ template "fullname" . }}
   labels:
     chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
 spec:
-  replicas: 1
+  replicas: {{ .Values.deployment.replicas }}
   template:
     metadata:
       labels:
-        app: {{ template "fullname" . }}-events
-      name: {{ template "fullname" . }}-events
+        app: {{ template "fullname" . }}
+      name: {{ template "fullname" . }}
     spec:
       containers:
-      - name: {{ .Chart.Name }}-events
+      - name: {{ .Chart.Name }}
         image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
         imagePullPolicy: {{ .Values.image.pullPolicy }}
         resources:
 {{ toYaml .Values.resources | indent 12 }}
+        ports:
+        - containerPort: 8125
```

Comment:
@seanknox I fully understand your concerns, but I believe it's a pretty general case.

IMO any big (>500 employees), multi-team, multi-unit, budget-aware company could be a good fit for this case. I believe we could call it a general case, because now I see K8s is being adopted even in bigger, conservative/slower companies, with many teams working on projects not having anything in common, but still wanting to enjoy benefits of unification of the deploy  and management process K8s could provide.

Also please don't forget the **flexibility** part.
Each team could maintain its own DataDog config, adding/changing application-specific checks without touching other teams and global cluster monitoring config. So it's not only about money. :wink: